sdk/eventhub/testhub/commands/sendReceive.ts

```diff
@@ -3,43 +3,46 @@
 import { CommandBuilder } from "yargs";
 import { EventHubClient, EventPosition, EventData, delay } from "@azure/event-hubs";
 import { log, setCurrentCommand, randomNumberFromInterval } from "../utils/util";
-import uuid from "uuid/v4";
+import { v4 as uuid } from "uuid";
 
 export const command = "send-receive";
 
-export const describe = "Sends a message to an event hub partition and receives the message " +
+export const describe =
```

Comment:
@chradek: Should these formatting changes be kept or reverted?

Reply:
Sorry...my editor runs the formatter automatically anytime a file is saved. I'm fine with keeping them, but can revert if it is making it too difficult to review the uuid changes.

Reply:
I like my PRs clean so I will revert them.  I think the only required change is the import statement?

Reply:
Yes, correct.

Reply:
Reverted